### PR TITLE
feat: Feature to add multiple labels using single command with comma seperated label names

### LINKED ISSUE #129: [Enhancement] Label list feature
## Description
As a developer,
I need to add a feature,
so that a user can command the bot to label an issue with a list of comma-seperated labels.

## Mocks
-N.A.-

### Update [Required]
- [ ] Create a sys-bot label command

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Added command to add multiple labels to issues at once. 

Fixes #129 

# Type of Change:
- Code
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
label_test

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by sammy1997** [comment]
Updated @sara-02 !

File: tests/test_github_functions.py [hunk 1]
```diff
@@ -141,3 +142,9 @@ def test_check_pr_template(self):
         response_wrong_template = check_pr_template("Test test",
                                                     'systers', 'sysbot-testing', '12')
         self.assertEqual(response_wrong_template, False)
+
+    def test_label_list_issue(self):
+        response_all_labelled = label_list_issue('systers', 'sysbot-test', 140, "@sys-bot label test-label, bug", "sammy1997")
```
Comment (sara-02) [inline tests/test_github_functions.py]: It will be good to have a test for `204` response as well.

**Comment by sara-02** [comment]
@sammy1997 Travis is complaining,
please add a new line here
https://travis-ci.org/systers/sysbot/builds/411300421?utm_source=github_status&utm_medium=notification
@ramitsawhney27 You can merge this once the Travis is green.

**Comment by sammy1997** [comment]
Checks passed @sara-02 @ramitsawhney27 !

**Pull request merged**